resources/views/ticket/index.blade.php

```diff
@@ -1 +1,82 @@
+                                    </span>
+                                </th>
+                                <th class="px-6 py-3 bg-gray-50 text-left">
+                                    <span class="text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider">
+                                        {{ __('temp.movie') }}
+                                    </span>
+                                </th>
+                                <th class="px-6 py-3 bg-gray-50 text-left">
+                                    <span class="text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider">
+                                        {{ __('temp.time') }}
+                                    </span>
+                                </th>
+                                <th class="px-6 py-3 bg-gray-50 text-left">
+                                    <span class="text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider">
+                                        {{ __('temp.who') }}
+                                    </span>
+                                </th>
+                                <th class="px-6 py-3 bg-gray-50 text-left">
+                                    <span class="text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider">
+                                        {{ __('temp.act') }}
+                                    </span>
+                                </th>
+                            </tr>
+                            </thead>
+
+                            <tbody class="bg-white divide-y divide-gray-200 divide-solid">
+                            @php
+                                $tickets = App\Models\Ticket::with('user')->get();   
+                            @endphp
+                            @foreach($tickets as $ticket)
+                                <tr class="bg-white">
+                                    <td class="px-6 py-4 whitespace-no-wrap text-sm leading-5 text-gray-900">
+                                        {{ $loop->iteration }}
+                                    </td>
+                                    <td class="px-6 py-4 whitespace-no-wrap text-sm leading-5 text-gray-900">
+                                        {{ $ticket->movie_name }} 
+                                    </td>
+                                    <td class="px-6 py-4 whitespace-no-wrap text-sm leading-5 text-gray-900">
+                                        {{ $ticket->time_show }}
+                                    <td class="px-6 py-4 whitespace-no-wrap text-sm leading-5 text-gray-900">
+                                        {{ $ticket->user->username }}
+                                    </td>
+                                    <td class="px-6 py-4 whitespace-no-wrap text-sm leading-5 text-red-500">
+                                        <a href="{{ route('tickets.edit',$ticket->id) }}"> {{ __('temp.edit') }} </a>
```

Comment:
```suggestion
                                        <a href="{{ route('tickets.edit', $ticket->id) }}"> {{ __('temp.edit') }} </a>
```
them khoang trang sau dau "," Fix tuong tu cac cho khac